salesforce_bulk/salesforce_bulk.py

```diff
@@ -380,12 +383,16 @@ def batch_status(self, job_id=None, batch_id=None, reload=False):
         resp, content = http.request(uri, headers=self.headers())
         self.check_status(resp, content)
 
-        tree = ET.fromstring(content)
-        result = {}
-        for child in tree:
-            result[re.sub("{.*?}", "", child.tag)] = child.text
 
-        self.batch_statuses[batch_id] = result
+        if self.headers()['Content-Type'] in ('application/json', 'JSON', 'application/xml; charset=UTF-8'):
```

Comment:
That's a good point. I suppose that should have been dropped down into an `elif self.headers()['Content-Type'] in ( 'application/xml; charset=UTF-8'):`